git-rename-master-branch-to-main

```diff
@@ -32,7 +32,7 @@ die() { >&2 printf %s\\n "$*" ; exit 1 ; }
 prompt_to_continue() {
 	printf %s "Continue? (yes/no) "
 	read answer
-	if [ "$answer" == "${answer#[Yy]}" ]; then
+	if [ "$answer" = "${answer#[Yy]}" ]; then
```

Comment:
this may not be needed or may cause issues elsewhere;

 on my linux machine i was getting syntax error here and my linter flagged it as, "== is undefined in POSIX shell"